pallets/parachain-staking/src/benchmarks.rs

```diff
@@ -17,72 +17,109 @@
 #![cfg(feature = "runtime-benchmarks")]
 
 //! Benchmarking
-use crate::{BalanceOf, Call, Config, Pallet, Range};
+use crate::{
+	BalanceOf, Call, CandidateBondChange, CandidateBondRequest, Config, DelegationChange,
+	DelegationRequest, Pallet, Range,
+};
 use frame_benchmarking::{account, benchmarks, impl_benchmark_test_suite};
 use frame_support::traits::{Currency, Get, OnFinalize, OnInitialize, ReservableCurrency};
 use frame_system::RawOrigin;
 use nimbus_primitives::EventHandler;
 use sp_runtime::{Perbill, Percent};
 use sp_std::{collections::btree_map::BTreeMap, vec::Vec};
 
-/// Default balance amount is minimum collator stake
-fn default_balance<T: Config>() -> BalanceOf<T> {
+/// Minimum collator candidate stake
+fn min_candidate_stk<T: Config>() -> BalanceOf<T> {
 	<<T as Config>::MinCollatorStk as Get<BalanceOf<T>>>::get()
 }
 
+/// Minimum delegator stake
+fn min_delegator_stk<T: Config>() -> BalanceOf<T> {
+	<<T as Config>::MinDelegatorStk as Get<BalanceOf<T>>>::get()
+}
+
 /// Create a funded user.
+/// Extra + min_candidate_stk is total minted funds
+/// Returns tuple (id, balance)
 fn create_funded_user<T: Config>(
 	string: &'static str,
 	n: u32,
 	extra: BalanceOf<T>,
-) -> T::AccountId {
+) -> (T::AccountId, BalanceOf<T>) {
 	const SEED: u32 = 0;
 	let user = account(string, n, SEED);
-	let default_balance = default_balance::<T>();
-	let total = default_balance + extra;
+	let min_candidate_stk = min_candidate_stk::<T>();
+	let total = min_candidate_stk + extra;
 	T::Currency::make_free_balance_be(&user, total);
 	T::Currency::issue(total);
-	user
+	(user, total)
 }
 
-/// Create a funded nominator. Base balance is MinCollatorStk == default_balance
-/// but the amount for the nomination is MinNominatorStk << MinCollatorStk => the rest + extra
-/// is free balance for the returned account.
+/// Create a funded nominator.
 fn create_funded_nominator<T: Config>(
```

Comment:
should this be `s/nominator/delegator/` as well?

Reply:
Yes, I haven't finished this term replacement yet, will do